src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -307,11 +290,10 @@ internal static new class Property
         /// <summary>
         /// Creates an uninitialized WebView.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
Please keep [EditorBrowsable(EditorBrowsableState.Never)] in DevelNUI branch.
When this patch is merged, you can remove it on Master branch with ACR.